addons/io_scene_gltf2/__init__.py

```diff
@@ -429,6 +443,15 @@ def execute(self, context):
         export_settings['gltf_binaryfilename'] = os.path.splitext(os.path.basename(
             bpy.path.ensure_ext(self.filepath,self.filename_ext)))[0] + '.bin'
 
+        import sys
+        user_extensions = []
+        for addon_name in bpy.context.preferences.addons.keys():
```

Comment:
In 2.79, [it is apparently called `bpy.context.user_preferences`](https://docs.blender.org/api/2.79/bpy.types.Context.html#bpy.types.Context.user_preferences). So see if this fixes the CI

````py
if bpy.app.version < (2, 80, 0):
    prefs = bpy.context.user_preferences
else:
    prefs = bpy.context.preferences
for addon_name in prefs.addons.keys():
````